src/containers/EventInfo.js

```diff
@@ -18,17 +16,10 @@ export class EventInfo extends Component {
     }
   }
 
-  componentWillReceiveProps (nextProps) {
-    if (this.props.event !== nextProps.event) {
-      this.setState({ event: nextProps.event })
-    }
-  }
-
   render () {
-    let { event } = this.state
     return (
       <Container className='event-info-container'>
-        <EventSummary event={event} />
+        <EventSummary event={this.props.event} />
```

Comment:
this is great! maybe I was overthinking things with setting the state instead of just using what is already in the store